tsup.config.ts

```diff
@@ -0,0 +1,11 @@
+import { defineConfig } from "tsup";
+
+export default defineConfig({
+  entry: ["src/index.ts", "src/core/index.ts", "src/utils/index.ts", "src/handlers/index.ts", "src/types/index.ts"],
+  splitting: false,
+  sourcemap: false,
```

Comment:
This also seems very useful
```suggestion
  sourcemap: true,
```

Reply:
Not source that sourcemap are useful except for stack traces and debug which should not be included in production. It is useful if the code is minimized, which is not the case here.

Reply:
See also https://tsup.egoist.dev/#generate-sourcemap-file

Reply:
> Not sure that sourcemap are useful except for stack traces and debug which should not be included in production. It is useful if the code is minimized, which is not the case here.

Stack traces aren't magically removed from production I see them all the time from packages